property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
+    result = None
```

Comment:
right, sorry...  I knew I missed something...  what if we re-factor it a bit and replace it with early exit?

```python
if fn.__name__ in cache:
    return cache[fn.__name__]
```